util/Order.h

```diff
@@ -613,6 +614,7 @@ class FO_COMMON_API ShipDesignOrder : public Order {
     void ExecuteImpl() const override;
 
     int m_design_id = INVALID_OBJECT_ID;
+    boost::uuids::uuid m_uuid;
```

Comment:
is this related to this pull request? it's not mentioned in or seemingly relevant to the description.